src/Speckle.Sdk/Transports/DiskTransport.cs

```diff
@@ -93,7 +93,7 @@ public void SaveObject(string id, string serializedObject)
     }
 
     SavedObjectCount++;
-    OnProgressAction?.Invoke(TransportName, SavedObjectCount);
+    OnProgressAction?.Invoke(new(ProgressEvent.DownloadObject, null, SavedObjectCount));
```

Comment:
I think we need some clarification on how to use `count/total`, I wasn't expecting one to be allowed to be null and not the other 🤔

Reply:
There's not really a guidance other than just use what's available to you.